app/views/internal/configs/show.html.erb

```diff
@@ -374,6 +374,16 @@
               <div class="alert alert-info">Used to limit the amount of articles a user can create within 30 seconds</div>
             </div>
 
+            <div class="form-group">
+              <%= f.label :rate_limit_organization_creation %>
+              <%= f.number_field :rate_limit_organization_creation,
+                                 class: "form-control",
+                                 value: SiteConfig.rate_limit_organization_creation,
+                                 min: 0,
```

Comment:
min should be 1 otherwise no org will be created I guess, right?

Reply:
Good call!